### ISSUE TITLE
npm warns of a critical security vulnerability

### ISSUE THREAD

**Issue by lacker** [webtorrent/webtorrent-cli#99]
The version of `webtorrent-cli` published to `npm` has a dependency on version `0.0.5` of the `open` library which has this security flaw: https://www.npmjs.com/advisories/663

You can see that the dependency exists by running:

```
$ npm view webtorrent-cli dependencies
{ clivas: '^0.2.0',
  'create-torrent': '^3.23.1',
  dlnacasts: '^0.1.0',
  ecstatic: '^3.0.0',
  executable: '^4.0.0',
  mime: '^2.1.0',
  minimist: '^1.2.0',
  moment: '^2.12.0',
  'network-address': '^1.1.0',
  open: '0.0.5',
  'parse-torrent': '^6.0.0',
  'prettier-bytes': '^1.0.3',
  'vlc-command': '^1.0.0',
  webtorrent: '0.x',
  winreg: '^1.0.1',
  'airplay-js': '^0.3.0',
  chromecasts: '^1.5.3',
  nodebmc: '0.0.7' }
```

However, when I look at the `package.json` in this repo, it seems like that dependency has already been updated to a secure one. So perhaps this module just needs a new version pushed?

**Comment by feross** [comment]
It looks like you're depending on an old version of `webtorrent-cli`. The latest version does not depend on `open`.

**Comment by lacker** [comment]
Looks like this dependency was fixed in the 1.12.4 release last week. Thanks for pushing a new one!

**Comment by feross** [comment]
Yep, no problem!
